lua/sqlite/defs.lua

```diff
@@ -6,14 +6,13 @@ local M = {}
 --- Load clib
 local clib = (function()
   local path, _
+  path, _ = luv.os_getenv "LIBSQLITE"
```

Comment:
should only read if vim.g.sqlite_clib_path is not set or is not vim, feels unnecessary call no?

Reply:
I moved it beneath as a fallback if im.g.sqlite_clib_path is not set